.eleventy.js

```diff
@@ -307,6 +307,15 @@ module.exports = eleventyConfig => {
     eleventyConfig.addShortcode("image", imageShortcode)
     // END, eleventy-img
 
+    /*
+     * Do not generate sitemap and sitemap index files for translation sites because we're
```

Comment:
I think we should at least generate a shallow site map for translation sites. That will help if people are searching in other languages.

Reply:
I added `multilingual: true` to 6 pages that have translations (did I miss some?) and updated the sitemap template to include only pages with this property on translation sites.

Note that only 3 of those 6 appear in the sitemap because the other 3 have `eleventyExcludeFromCollections: true`.

Pages in the sitemap for translation sites:

* https://es.eslint.org/donate/
* https://es.eslint.org/languages/
* https://es.eslint.org/team/

Pages _not_ in the sitemap because of `eleventyExcludeFromCollections: true` (was that intentional?):

* https://es.eslint.org/
* https://es.eslint.org/sponsors/
* https://es.eslint.org/branding/